src/app/components/pages/UserProfile.jsx

```diff
@@ -434,20 +444,27 @@ export default class UserProfile extends React.Component {
             </div>
          </div>;
 
-        const {name, location, about, website, cover_image} = normalizeProfile(account);
+        const {name, location, about, website, cover_image, cover_image_mobile} = normalizeProfile(account);
         const website_label = website ? website.replace(/^https?:\/\/(www\.)?/, '').replace(/\/$/, '') : null
 
         let cover_image_style = {}
         if(cover_image) {
             cover_image_style = {backgroundImage: "url(" + proxifyImageUrl(cover_image, '2048x512') + ")"}
         }
 
+        let cover_image_mobile_style = {}
+        if(cover_image_mobile) {
+            cover_image_mobile_style = {backgroundImage: "url(" + proxifyImageUrl(cover_image_mobile, '640x512') + ")"}
+        }
+
+        const showMobileCover = this.state.showMobileCover;
+
         return (
             <div className="UserProfile">
 
                 <div className="UserProfile__banner row expanded">
 
-                    <div className="column" style={cover_image_style}>
+                    <div className="column" style={showMobileCover ? cover_image_mobile_style : cover_image_style}>
```

Comment:
What styles are applied here if `showMobileCover` is true but there is no `cover_image_mobile` ... won't `cover_image_mobile_style` be an empty object?